extensions/src/ACRE2Arma/acre/acre.cpp

```diff
@@ -77,9 +90,29 @@ void __stdcall RVExtension(char *output, int outputSize, const char *function) {
         _threaded = true;
     }
 
-    acre::controller::get().call(command, _args, result, _threaded);
-    if (result.length() > 0) {
-        sprintf_s(output, outputSize, "%s", result.c_str());
+    acre::controller::get().call(command, _args, result_string, result_dummy, _threaded);
+    if (result_string.length() > 0) {
+        sprintf_s(output, outputSize, "%s", result_string.c_str());
     }
     EXTENSION_RETURN();
 }
+
+int __stdcall RVExtensionArgs(char* output, int outputSize, const char* function, const char** argv, int argc) {
+    ZERO_OUTPUT();
+    if (argc < 2) { EXTENSION_RETURN_VALUE(-2); } // callExtension ["calla", ["extension.dll", "func", ...]]
+
+    std::string command(function);
+    acre::arguments _args(argv, argc);
+
+    //LOG(INFO) << "RVExtensionArgs [" << function << " - " << _args.to_string() << "]";
+    if (command != "process_signal") { EXTENSION_RETURN_VALUE(-3); } // only supported command for now
+
+    std::string result_string = "";
```

Comment:
```suggestion
    std::string result_string;
```

no need to initialize as an empty string